ncbi_genome_download/core.py

```diff
@@ -153,12 +154,78 @@ def choices(self):
                          ['full_url', 'local_file', 'expected_checksum', 'symlink_path'])
 
 
-# pylint and I disagree on code style here. Shut up, pylint.
-# pylint: disable=too-many-locals
+def argument_parser(version=None):
+    """Argument parser for ncbi-genome-download"""
+    parser = argparse.ArgumentParser()
+    parser.add_argument('group',
+                        #choices=EDefaults.TAXONOMIC_GROUPS.choices, # Moved choice checking to method
```

Comment:
Please never commit commented-out code. That's why we have version control, after all. :wink: